bucket/strokesplus.net.json

```diff
@@ -0,0 +1,39 @@
+{
+    "version": "0.5.6.6",
+    "description": "Mouse gesture recognition utility for Windows which allows you to create powerful mouse gestures that save you time.",
+    "homepage": "https://www.strokesplus.net",
+    "license": {
+        "identifier": "Freeware",
+        "url": "https://www.strokesplus.net/FAQ"
+    },
+    "architecture": {
+        "64bit": {
+            "url": "https://cdn.strokesplus.net/files/StrokesPlus.net_Portable_0.5.6.6.zip",
+            "hash": "b1c82604910676bc7bb3cd43b02a278bc01f51b212a6c180aab59f0a6ec86e9f"
+        },
+        "32bit": {
+            "url": "https://cdn.strokesplus.net/files/StrokesPlus.net_Portable_0.5.6.6.zip",
+            "hash": "b1c82604910676bc7bb3cd43b02a278bc01f51b212a6c180aab59f0a6ec86e9f"
+        }
+    },

```

Comment:
Please remove the architecture block - there is no difference between the 64bit and 32bit url and hash. The app can run on 32bit computers too.